pydeseq2/DeseqStats.py

```diff
@@ -28,8 +29,15 @@ class DeseqStats:
     dds : DeseqDataSet
         DeseqDataSet for which dispersion and LFCs were already estimated.
 
+    contrast : list[str] or None
```

Comment:
from this description I am not sure what contrast actually is (just the format of it) and when to use it instead of None. Could you add a short description?

Reply:
I've expanded the description of this argument : [8c2197a](https://github.com/owkin/PyDESeq2/pull/32/commits/8c2197a349d45c5a089b23981c58e0f270c3c66a)

Let me know if this helps, or it is still a bit obscure

Reply:
yes, thanks!